server/commands/areas.py

```diff
@@ -708,3 +709,15 @@ def ooc_cmd_lights(client, arg):
             pos = client.area.pos_dark
         c.send_command("BN", bg, pos)
     client.send_ooc(f"This area is {stat} dark.")
+
+def ooc_cmd_gethubs(client, arg):
+    """
+    Show information about all areas of all hubs.
+    Usage: /gethubs
+    """
+    if not client.is_mod and not (client in client.area.area_manager.owners) and client.char_id != -1:
+        if client.blinded:
+            raise ClientError('You are blinded!')
+        if not client.area.area_manager.can_getareas:
```

Comment:
Wait you can bypass all protections ever by using /gethubs from a hub that allows getareas, this is not a good solution. can_getareas should be checked elsewhere.

Reply:
So in an area where can_getareas (or also can_getarea?) is False, you don't want that users could check the client list of that area, right?

Reply:
Yes